Automattic/newspack-plugin
# feat: (IA) dashboard icons

### All Submissions:


### Changes proposed in this Pull Request:

* Adds support for all icons needed for new information architecture.
* Custom icons (not found [Gutenberg Icons](https://wordpress.github.io/gutenberg/?path=/story/icons-icon--library)) in have been converted to React components using `@wordpress/primitives` 

### How to test the changes in this Pull Request:

1. Checkout this branch and build assets i.e. `git checkout feat/ia-dash-icons && npm run build`
2. Navigate to new app - /wp-admin/admin.php?page=newspack#/
3. Observe you see the below:

![Screenshot 2024-03-25 at 22 19 44](https://github.com/Automattic/newspack-plugin/assets/3676975/eb2da9b0-b4f7-4bee-bf71-3934c513c1e8)

### Other information:
